package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "ash.ts",
-  "private": true,
```

Comment:
I used `"private": true` because I'm not an owner of `ash.ts` npm package. There is no `ash.ts` package and it cannot be created in npm because of similarity to `ash-ts`. I'm the owner of `@ash.ts` namespace.

Reply:
Also look at package.json files inside packages/*/ - they are public and published.